### PR TITLE
feat: UNT-T1048: Added custom header component

### PR DESCRIPTION
**Pull request by stutidyer-simformsolutions**
Created header component and used it on homescreen

### CODE REVIEW

File: app/components/CustomHeader/CustomHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { View } from 'native-base';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import React, { FC } from 'react';
```
Comment (Krunal-K-SimformSolutions) [inline app/components/CustomHeader/CustomHeader.tsx]: FC import as a type. 
do tsconfig.json in add below key to throwing error when not define only-type import
"isolatedModules": true,
 "preserveValueImports": true,
"esModuleInterop": true

File: app/components/CustomHeader/CustomHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          <Image source={logoIcon} style={styles.logoImageStyle} />
+        )}
+        {rightIconName ? (
+          <Icon
+            name={rightIconName}
+            size={25}
+            color={Color.lightGreen}
+            onPress={onRightPress}
+          />
+        ) : null}
```
Comment (Krunal-K-SimformSolutions) [inline app/components/CustomHeader/CustomHeader.tsx]: remove null

File: app/components/CustomHeader/CustomHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      </View>
+      {isSearchIconVisible ? (
+        <Icon
+          name={'magnify'}
+          size={25}
+          color={Color.lightGreen}
+          onPress={onSearchPress}
+          style={styles.searchImageStyle}
+        />
+      ) : null}
```
Comment (Krunal-K-SimformSolutions) [inline app/components/CustomHeader/CustomHeader.tsx]: remove null

Comment (stutidyer-simformsolutions) [reply app/components/CustomHeader/CustomHeader.tsx]: @Krunal-K-SimformSolutions  What should I add an empty view for condition?

File: app/components/CustomHeader/CustomHeader.tsx (hunk 2 above)
Comment (stutidyer-simformsolutions) [reply app/components/CustomHeader/CustomHeader.tsx]: @Krunal-K-SimformSolutions  What should I add an empty view for condition?

Comment (Krunal-K-SimformSolutions) [reply app/components/CustomHeader/CustomHeader.tsx]: Nope, use logical && instead of ternary operator.

**Pull request merged**